README.md

```diff
@@ -322,6 +355,23 @@ A style guide for the [HTML Template Language](https://docs.adobe.com/docs/en/ht
       <sly data-sly-call="${teaserTemplates.teaserSmall @ teaserModel=teaser}"></sly>
     </sly>
     ```
+
+- [4.11](#4.11) <a name='4.11'></a> **Avoid using data-sly-test to set arbitrary variable bindings**
```

Comment:
maybe we should elaborate on why?
one case I've seen is people using several data-sly-test on a single <sly> tag without realising that a single _data-sly-test_ that returns a false|null value will end the evaluation for that tag